readme.md

```diff
@@ -40,6 +40,12 @@ re-initializing the database.
 If existing, the file `settings.local.py` at the root of the project will
 override the default `settings.py` configuration.
 
+Do not forget to create the folder dedicated to statics, by default:
```

Comment:
This should be generated with `make install`.

Reply:
Do you recommend to run `make install` for devs too?

Reply:
Mhh… not sure. Let's keep it like you did for now.